src/ResourceManager/RedisCache/Commands.RedisCache/Commands/SetAzureRedisCacheDiagnostics.cs

```diff
@@ -40,15 +40,14 @@ public override void ExecuteCmdlet()
         {
             Utility.ValidateResourceGroupAndResourceName(ResourceGroupName, Name);
             ResourceGroupName = CacheClient.GetResourceGroupNameIfNotProvided(ResourceGroupName, Name);
-            string storageAccountName = GetStorageAccountName(StorageAccountId);
 
             RedisCacheAttributes cache = new RedisCacheAttributes(CacheClient.GetCache(ResourceGroupName, Name), ResourceGroupName);
             ConfirmAction(
               string.Format(Resources.SetRedisCacheDiagnostics, Name),
               Name,
               () => 
               {
-                  CacheClient.SetDiagnostics(cache.Id, storageAccountName);
+                  CacheClient.SetDiagnostics(cache.Id, StorageAccountId);
```

Comment:
Are the Redis folks OK with these changes?

Reply:
No idea. I had added them to this, but removed them after a week of no responses. Do you want me to ping them via email?

Reply:
Send an email to @SiddharthChatrolaMs

Reply:
I split out their code changes to a separate PR. This can be merged later when they update their service. https://github.com/Azure/azure-powershell/pull/5601